### ISSUE TITLE
Package Proposal: Humanize

### ISSUE THREAD

**Issue by jeanluc243** [fluttercommunity/community#39]
# Package Proposal: Humanize

**Dependency name (as used in *pubspec.yaml*):** humanize
**Current pub.dev link:** https://pub.dev/packages/humanize/
**Current Git repository link:** https://github.com/jeanluc243/humanize
**Description:** A package for translating numbers and dates into a human readable format.
**Current maintainer:** Jean Luc Kabulu (https://github.com/jeanluc243/humanize / jeanluc.kabulu@gmail.com )
**Needs new maintainer after transfer:** NO
**Reason for transfer:**  i'd like to recognize the package as a work of the community.

**Comment by jeroen-meijer** [comment]
Hi there,

Thanks for the package proposal.
Due to recent events within the community and the world overall, we're currently quite busy and will not be able to respond to most package proposals and other community inquiries on GitHub as well as you might expect. Sorry about that. We'll keep you up to date.

Thank you for understanding.

Best regards,
Jay (Flutter Community GitHub admin).

**Comment by jeroen-meijer** [comment]
Hi there,

Thanks again for submitting the package. 👍 
It looks quite convenient.

Unfortunately, it doesn't seem like it's quite finished, so doing a full review at this time is redundant. Let me explain.

# Rejected

I like the idea of a package to streamline ordinals and other string representations of numbers - I'd use this in several projects! 😄

However, after looking at the codebase, it simply doesn't seem finished.
That's why reviewing the package doesn't make sense to me.

Don't worry, you can resubmit the package after finishing it and doing some improvements. Please see the following points to keep in mind.

## Points of improvement

- There are multiple functions that aren't implemented but are exposed. It's always a bad idea to expose unimplemented functionality because users will try to use your function and not know what's going on. For example:
```dart
/// Convert an integer to a string containing commas every three digits.
/// For example, 3000 becomes '3,000' and 45000 becomes '45,000'.
String intComma(int value) {
  return "";
}

/// Convert a large integer to a friendly text representation. Works best
/// for numbers over 1 million. For example, 1000000 becomes '1.0 million',
/// 1200000 becomes '1.2 million' and '1200000000' becomes '1.2 billion'.
String intWord(int value) {
  return "";
}
```
Please remove those functions or implement them, and bump the version of your package with a breaking change/MAJOR revision number (for example, `1.0.0`). [Read this for more info.](https://semver.org)

- In `/test/humanize_test.dart`, the `Calculator` class isn't found.
In fact, it seems like there are no tests at all, and that the default test was used. Tests are important, and for a library like this, not difficult to implement.

- There are several pull requests and issues on this package's Git repository that have been unanswered or otherwise unattended. These include some fixes and feature improvements that look quite important for this package. As a package maintainer of the Flutter Community, it's your responsibility to respond to PRs and issues in a timely fashion. If you can't keep up or don't have time anymore, that's totally fine - you always have the option to transfer ownership of this package to another maintainer.

## In conclusion

Though your package looks promising for simple number formatting, it doesn't look finished yet. That's why reviewing at this time does not make sense to me. Please look at the feedback above and feel free to resubmit this package as soon as the above points of improvement have been taken care of.

Thanks again, and good luck! 👍

Closing.
